sky/shell/platform_view.h

```diff
@@ -56,19 +63,28 @@ class PlatformView {
 
   virtual bool ContextMakeCurrent() = 0;
 
+  virtual bool ResourceContextMakeCurrent() = 0;
+
   virtual bool SwapBuffers() = 0;
 
   virtual SkISize GetSize();
 
   virtual void Resize(const SkISize& size);
 
  protected:
-  explicit PlatformView(const Config& config, SurfaceConfig surface_config);
-
   Config config_;
   SurfaceConfig surface_config_;
+
+  Shell& shell_;
```

Comment:
That was in the old shell view that I mushed into the platform view. I think it was there as a way to indicate that an instantiated instance of the shell needed to be provided before a platform view could be created. I am going to get rid of it.